src/state/nftMarket/helpers.ts

```diff
@@ -201,3 +212,84 @@ export const fetchWalletTokenIdsForCollections = async (
   const walletNfts = await Promise.all(walletNftPromises)
   return walletNfts.flat()
 }
+
+/**
+ * Get the 8 most recently listed NFTs
+ * @returns NftTokenSg[]
+ */
+export const getLatestListedNfts = async (): Promise<NftTokenSg[]> => {
```

Comment:
I think we should simply have a nfts helper that takes in orderby, first, orderDirection, and where parameters. That way we can reuse for anything.